TYP:  mypy reports `error: "TypeAliasType" not callable` when creating arrays with `numpy.ma.masked_array` since numpy v2.5

### Describe the issue:

Running the code example below through mypy with the command `pixi x --with numpy=2.5 mypy --verbose myproblem.py` gives the following error:
```
LOG:  Mypy Version:           2.1.0
...
myproblem.py:3: error: "TypeAliasType" not callable  [operator]
```

It looks like this happens because `masked_array` is defined as a type alias in this file:
https://github.com/numpy/numpy/blob/v2.5.0/numpy/ma/core.pyi#L2851
while the code works fine because it is also defined as an alias in
https://github.com/numpy/numpy/blob/v2.5.0/numpy/ma/core.py#L6856
but `mypy` doesn't seem to understand that.

### Reproduce the code example:

```python
import numpy as np

a = np.ma.masked_array([1])
```

### Error message:

```shell
myproblem.py:3: error: "TypeAliasType" not callable  [operator]
```

### Python and NumPy Versions:

Python 3.14
Numpy 2.5

### Type-checker version and settings:

Mypy 2.1.0

No specific configuration for mypy. Full output of the command `pixi x --with numpy=2.5 mypy --verbose myproblem.py`:

<details>

```
LOG:  Mypy Version:           2.1.0
LOG:  Config File:            Default
LOG:  Configured Executable:  /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/bin/python3.14
LOG:  Current Executable:     /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/bin/python3.14
LOG:  Cache Dir:              .mypy_cache
LOG:  Compiled:               True
LOG:  Exclude:                []
LOG:  Found source:           BuildSource(path='myproblem.py', module='myproblem', has_text=False, base_dir='/home/bandela/src/esmvalgroup/esmvaltool', followed=False)
LOG:  Could not load cache for myproblem: myproblem.meta.ff
LOG:  Metadata not found for myproblem
LOG:  Parsing myproblem.py (myproblem)
LOG:  Metadata fresh for numpy: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/__init__.pyi
LOG:  Metadata fresh for builtins: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/builtins.pyi
LOG:  Metadata fresh for numpy._core._internal: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/_internal.pyi
LOG:  Metadata fresh for numpy._typing._char_codes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_char_codes.py
LOG:  Metadata fresh for numpy._typing._extended_precision: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_extended_precision.py
LOG:  Metadata fresh for numpy._core.records: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/records.pyi
LOG:  Metadata fresh for numpy._core.function_base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/function_base.pyi
LOG:  Metadata fresh for numpy._core.fromnumeric: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/fromnumeric.pyi
LOG:  Metadata fresh for numpy._core._asarray: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/_asarray.pyi
LOG:  Metadata fresh for numpy._core._type_aliases: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/_type_aliases.pyi
LOG:  Metadata fresh for numpy._core._ufunc_config: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/_ufunc_config.pyi
LOG:  Metadata fresh for numpy._core.arrayprint: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/arrayprint.pyi
LOG:  Metadata fresh for numpy._core.einsumfunc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/einsumfunc.pyi
LOG:  Metadata fresh for numpy._core.getlimits: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/getlimits.pyi
LOG:  Metadata fresh for numpy._core.memmap: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/memmap.pyi
LOG:  Metadata fresh for numpy._core.multiarray: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/multiarray.pyi
LOG:  Metadata fresh for numpy._core.numeric: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/numeric.pyi
LOG:  Metadata fresh for numpy._core.numerictypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/numerictypes.pyi
LOG:  Metadata fresh for numpy._core.shape_base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/shape_base.pyi
LOG:  Metadata fresh for numpy.lib.scimath: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/scimath.pyi
LOG:  Metadata fresh for numpy.lib._arraypad_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_arraypad_impl.pyi
LOG:  Metadata fresh for numpy.lib._arraysetops_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_arraysetops_impl.pyi
LOG:  Metadata fresh for numpy.lib._function_base_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_function_base_impl.pyi
LOG:  Metadata fresh for numpy.lib._histograms_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_histograms_impl.pyi
LOG:  Metadata fresh for numpy.lib._index_tricks_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_index_tricks_impl.pyi
LOG:  Metadata fresh for numpy.lib._nanfunctions_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_nanfunctions_impl.pyi
LOG:  Metadata fresh for numpy.lib._npyio_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_npyio_impl.pyi
LOG:  Metadata fresh for numpy.lib._polynomial_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_polynomial_impl.pyi
LOG:  Metadata fresh for numpy.lib._shape_base_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_shape_base_impl.pyi
LOG:  Metadata fresh for numpy.lib._stride_tricks_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_stride_tricks_impl.pyi
LOG:  Metadata fresh for numpy.lib._twodim_base_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_twodim_base_impl.pyi
LOG:  Metadata fresh for numpy.lib._type_check_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_type_check_impl.pyi
LOG:  Metadata fresh for numpy.lib._ufunclike_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_ufunclike_impl.pyi
LOG:  Metadata fresh for numpy.lib._utils_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_utils_impl.pyi
LOG:  Metadata fresh for numpy.__config__: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/__config__.pyi
LOG:  Metadata fresh for numpy._pytesttester: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_pytesttester.pyi
LOG:  Metadata fresh for numpy._typing: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/__init__.py
LOG:  Metadata fresh for numpy._array_api_info: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_array_api_info.pyi
LOG:  Metadata fresh for collections.abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/collections/abc.pyi
LOG:  Metadata fresh for numpy.char: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/char/__init__.pyi
LOG:  Metadata fresh for numpy.core: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/__init__.pyi
LOG:  Metadata fresh for numpy.ctypeslib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/ctypeslib/__init__.pyi
LOG:  Metadata fresh for numpy.dtypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/dtypes.pyi
LOG:  Metadata fresh for numpy.exceptions: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/exceptions.pyi
LOG:  Metadata fresh for numpy.f2py: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/f2py/__init__.pyi
LOG:  Metadata fresh for numpy.fft: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/fft/__init__.pyi
LOG:  Metadata fresh for numpy.lib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/__init__.pyi
LOG:  Metadata fresh for numpy.linalg: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/linalg/__init__.pyi
LOG:  Metadata fresh for numpy.ma: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/ma/__init__.pyi
LOG:  Metadata fresh for numpy.polynomial: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/__init__.pyi
LOG:  Metadata fresh for numpy.random: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/__init__.pyi
LOG:  Metadata fresh for numpy.rec: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/rec/__init__.pyi
LOG:  Metadata fresh for numpy.strings: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/strings/__init__.pyi
LOG:  Metadata fresh for numpy.testing: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/testing/__init__.pyi
LOG:  Metadata fresh for numpy.typing: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/typing/__init__.pyi
LOG:  Metadata fresh for numpy.matlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/matlib.pyi
LOG:  Metadata fresh for numpy.matrixlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/matrixlib/__init__.pyi
LOG:  Metadata fresh for numpy.version: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/version.pyi
LOG:  Metadata fresh for numpy._expired_attrs_2_0: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_expired_attrs_2_0.pyi
LOG:  Metadata fresh for numpy._globals: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_globals.pyi
LOG:  Metadata fresh for ctypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/ctypes/__init__.pyi
LOG:  Metadata fresh for datetime: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/datetime.pyi
LOG:  Metadata fresh for inspect: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/inspect.pyi
LOG:  Metadata fresh for sys: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/sys/__init__.pyi
LOG:  Metadata fresh for abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/abc.pyi
LOG:  Metadata fresh for decimal: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/decimal.pyi
LOG:  Metadata fresh for fractions: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/fractions.pyi
LOG:  Metadata fresh for types: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/types.pyi
LOG:  Metadata fresh for uuid: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/uuid.pyi
LOG:  Metadata fresh for typing: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/typing.pyi
LOG:  Metadata fresh for _typeshed: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_typeshed/__init__.pyi
LOG:  Metadata fresh for typing_extensions: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/typing_extensions.pyi
LOG:  Metadata fresh for _ast: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_ast.pyi
LOG:  Metadata fresh for _sitebuiltins: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_sitebuiltins.pyi
LOG:  Metadata fresh for _collections_abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_collections_abc.pyi
LOG:  Metadata fresh for io: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/io.pyi
LOG:  Metadata fresh for os: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/os/__init__.pyi
LOG:  Metadata fresh for numpy._core: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/__init__.pyi
LOG:  Metadata fresh for re: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/re.pyi
LOG:  Metadata fresh for numpy._typing._array_like: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_array_like.py
LOG:  Metadata fresh for contextlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/contextlib.pyi
LOG:  Metadata fresh for functools: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/functools.pyi
LOG:  Metadata fresh for numpy._typing._ufunc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_ufunc.pyi
LOG:  Metadata fresh for numpy._core.umath: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/umath.pyi
LOG:  Metadata fresh for numpy.lib._scimath_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_scimath_impl.pyi
LOG:  Metadata fresh for numpy.lib._datasource: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_datasource.pyi
LOG:  Metadata fresh for zipfile: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/zipfile/__init__.pyi
LOG:  Metadata fresh for enum: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/enum.pyi
LOG:  Metadata fresh for numpy._typing._dtype_like: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_dtype_like.py
LOG:  Metadata fresh for numpy._typing._nbit: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_nbit.py
LOG:  Metadata fresh for numpy._typing._nbit_base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_nbit_base.pyi
LOG:  Metadata fresh for numpy._typing._nested_sequence: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_nested_sequence.py
LOG:  Metadata fresh for numpy._typing._scalars: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_scalars.py
LOG:  Metadata fresh for numpy._typing._shape: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_typing/_shape.py
LOG:  Metadata fresh for collections: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/collections/__init__.pyi
LOG:  Metadata fresh for numpy._core.defchararray: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/defchararray.pyi
LOG:  Metadata fresh for numpy.core._dtype: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/_dtype.pyi
LOG:  Metadata fresh for numpy.core._dtype_ctypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/_dtype_ctypes.pyi
LOG:  Metadata fresh for numpy.core._internal: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/_internal.pyi
LOG:  Metadata fresh for numpy.core.arrayprint: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/arrayprint.pyi
LOG:  Metadata fresh for numpy.core.defchararray: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/defchararray.pyi
LOG:  Metadata fresh for numpy.core.einsumfunc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/einsumfunc.pyi
LOG:  Metadata fresh for numpy.core.fromnumeric: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/fromnumeric.pyi
LOG:  Metadata fresh for numpy.core.function_base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/function_base.pyi
LOG:  Metadata fresh for numpy.core.getlimits: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/getlimits.pyi
LOG:  Metadata fresh for numpy.core.multiarray: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/multiarray.pyi
LOG:  Metadata fresh for numpy.core.numeric: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/numeric.pyi
LOG:  Metadata fresh for numpy.core.numerictypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/numerictypes.pyi
LOG:  Metadata fresh for numpy.core.overrides: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/overrides.pyi
LOG:  Metadata fresh for numpy.core.records: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/records.pyi
LOG:  Metadata fresh for numpy.core.shape_base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/shape_base.pyi
LOG:  Metadata fresh for numpy.core.umath: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/core/umath.pyi
LOG:  Metadata fresh for numpy.ctypeslib._ctypeslib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/ctypeslib/_ctypeslib.pyi
LOG:  Metadata fresh for numpy.f2py.f2py2e: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/f2py/f2py2e.pyi
LOG:  Metadata fresh for numpy.fft._helper: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/fft/_helper.pyi
LOG:  Metadata fresh for numpy.fft._pocketfft: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/fft/_pocketfft.pyi
LOG:  Metadata fresh for numpy.lib._array_utils_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_array_utils_impl.pyi
LOG:  Metadata fresh for numpy.lib._arrayterator_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_arrayterator_impl.pyi
LOG:  Metadata fresh for numpy.lib._format_impl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_format_impl.pyi
LOG:  Metadata fresh for numpy.lib._iotools: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_iotools.pyi
LOG:  Metadata fresh for numpy.lib._version: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/_version.pyi
LOG:  Metadata fresh for numpy.lib.array_utils: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/array_utils.pyi
LOG:  Metadata fresh for numpy.lib.format: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/format.pyi
LOG:  Metadata fresh for numpy.lib.introspect: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/introspect.pyi
LOG:  Metadata fresh for numpy.lib.mixins: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/mixins.pyi
LOG:  Metadata fresh for numpy.lib.npyio: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/npyio.pyi
LOG:  Metadata fresh for numpy.lib.stride_tricks: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/lib/stride_tricks.pyi
LOG:  Metadata fresh for numpy.linalg._linalg: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/linalg/_linalg.pyi
LOG:  Metadata fresh for numpy.linalg._umath_linalg: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/linalg/_umath_linalg.pyi
LOG:  Metadata fresh for numpy.ma.core: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/ma/core.pyi
LOG:  Metadata fresh for numpy.ma.extras: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/ma/extras.pyi
LOG:  Metadata fresh for numpy.polynomial.chebyshev: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/chebyshev.pyi
LOG:  Metadata fresh for numpy.polynomial.hermite: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/hermite.pyi
LOG:  Metadata fresh for numpy.polynomial.hermite_e: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/hermite_e.pyi
LOG:  Metadata fresh for numpy.polynomial.laguerre: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/laguerre.pyi
LOG:  Metadata fresh for numpy.polynomial.legendre: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/legendre.pyi
LOG:  Metadata fresh for numpy.polynomial.polynomial: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/polynomial.pyi
LOG:  Metadata fresh for numpy.random._generator: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/_generator.pyi
LOG:  Metadata fresh for numpy.random._mt19937: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/_mt19937.pyi
LOG:  Metadata fresh for numpy.random._pcg64: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/_pcg64.pyi
LOG:  Metadata fresh for numpy.random._philox: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/_philox.pyi
LOG:  Metadata fresh for numpy.random._sfc64: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/_sfc64.pyi
LOG:  Metadata fresh for numpy.random.bit_generator: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/bit_generator.pyi
LOG:  Metadata fresh for numpy.random.mtrand: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/random/mtrand.pyi
LOG:  Metadata fresh for numpy._core.strings: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/_core/strings.pyi
LOG:  Metadata fresh for numpy.testing._private.extbuild: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/testing/_private/extbuild.pyi
LOG:  Metadata fresh for numpy.testing._private.utils: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/testing/_private/utils.pyi
LOG:  Metadata fresh for numpy.testing._private: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/testing/_private/__init__.pyi
LOG:  Metadata fresh for numpy.testing.overrides: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/testing/overrides.pyi
LOG:  Metadata fresh for unittest: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/__init__.pyi
LOG:  Metadata fresh for numpy.matrixlib.defmatrix: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/matrixlib/defmatrix.pyi
LOG:  Metadata fresh for ctypes._endian: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/ctypes/_endian.pyi
LOG:  Metadata fresh for _ctypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_ctypes.pyi
LOG:  Metadata fresh for time: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/time.pyi
LOG:  Metadata fresh for dis: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/dis.pyi
LOG:  Metadata fresh for annotationlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/annotationlib.pyi
LOG:  Metadata fresh for _typeshed.importlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_typeshed/importlib.pyi
LOG:  Metadata fresh for sys._monitoring: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/sys/_monitoring.pyi
LOG:  Metadata fresh for numbers: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/numbers.pyi
LOG:  Metadata fresh for _decimal: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_decimal.pyi
LOG:  Metadata fresh for importlib.machinery: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/machinery.pyi
LOG:  Metadata fresh for dataclasses: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/dataclasses.pyi
LOG:  Metadata fresh for warnings: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/warnings.pyi
LOG:  Metadata fresh for ast: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/ast.pyi
LOG:  Metadata fresh for _io: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_io.pyi
LOG:  Metadata fresh for os.path: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/os/path.pyi
LOG:  Metadata fresh for subprocess: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/subprocess.pyi
LOG:  Metadata fresh for resource: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/resource.pyi
LOG:  Metadata fresh for sre_compile: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/sre_compile.pyi
LOG:  Metadata fresh for sre_constants: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/sre_constants.pyi
LOG:  Metadata fresh for contextvars: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/contextvars.pyi
LOG:  Metadata fresh for pathlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/pathlib/__init__.pyi
LOG:  Metadata fresh for zipfile._path: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/zipfile/_path/__init__.pyi
LOG:  Metadata fresh for numpy.f2py.__version__: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/f2py/__version__.pyi
LOG:  Metadata fresh for numpy.f2py.auxfuncs: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/f2py/auxfuncs.pyi
LOG:  Metadata fresh for argparse: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/argparse.pyi
LOG:  Metadata fresh for pprint: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/pprint.pyi
LOG:  Metadata fresh for numpy.polynomial._polybase: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/_polybase.pyi
LOG:  Metadata fresh for numpy.polynomial._polytypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/_polytypes.pyi
LOG:  Metadata fresh for numpy.polynomial.polyutils: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/polynomial/polyutils.pyi
LOG:  Metadata fresh for threading: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/threading.pyi
LOG:  Metadata fresh for unittest.case: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/case.pyi
LOG:  Metadata fresh for unittest.async_case: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/async_case.pyi
LOG:  Metadata fresh for unittest.loader: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/loader.pyi
LOG:  Metadata fresh for unittest.main: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/main.pyi
LOG:  Metadata fresh for unittest.result: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/result.pyi
LOG:  Metadata fresh for unittest.runner: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/runner.pyi
LOG:  Metadata fresh for unittest.signals: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/signals.pyi
LOG:  Metadata fresh for unittest.suite: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/suite.pyi
LOG:  Metadata fresh for opcode: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/opcode.pyi
LOG:  Metadata fresh for importlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/__init__.pyi
LOG:  Metadata fresh for importlib._bootstrap: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/_bootstrap.pyi
LOG:  Metadata fresh for importlib._bootstrap_external: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/_bootstrap_external.pyi
LOG:  Metadata fresh for _warnings: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_warnings.pyi
LOG:  Metadata fresh for codecs: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/codecs.pyi
LOG:  Metadata fresh for posixpath: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/posixpath.pyi
LOG:  Metadata fresh for sre_parse: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/sre_parse.pyi
LOG:  Metadata fresh for _contextvars: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_contextvars.pyi
LOG:  Metadata fresh for pathlib.types: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/pathlib/types.pyi
LOG:  Metadata fresh for numpy.f2py.cfuncs: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/numpy/f2py/cfuncs.pyi
LOG:  Metadata fresh for _thread: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_thread.pyi
LOG:  Metadata fresh for unittest._log: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/unittest/_log.pyi
LOG:  Metadata fresh for logging: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/logging/__init__.pyi
LOG:  Metadata fresh for asyncio.events: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/events.pyi
LOG:  Metadata fresh for importlib.abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/abc.pyi
LOG:  Metadata fresh for _frozen_importlib: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_frozen_importlib.pyi
LOG:  Metadata fresh for _frozen_importlib_external: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_frozen_importlib_external.pyi
LOG:  Metadata fresh for _codecs: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_codecs.pyi
LOG:  Metadata fresh for genericpath: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/genericpath.pyi
LOG:  Metadata fresh for signal: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/signal.pyi
LOG:  Metadata fresh for string: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/string/__init__.pyi
LOG:  Metadata fresh for asyncio: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/__init__.pyi
LOG:  Metadata fresh for concurrent.futures: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/futures/__init__.pyi
LOG:  Metadata fresh for asyncio.base_events: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/base_events.pyi
LOG:  Metadata fresh for asyncio.futures: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/futures.pyi
LOG:  Metadata fresh for asyncio.protocols: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/protocols.pyi
LOG:  Metadata fresh for asyncio.tasks: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/tasks.pyi
LOG:  Metadata fresh for asyncio.transports: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/transports.pyi
LOG:  Metadata fresh for ssl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/ssl.pyi
LOG:  Metadata fresh for _asyncio: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_asyncio.pyi
LOG:  Metadata fresh for socket: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/socket.pyi
LOG:  Metadata fresh for importlib._abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/_abc.pyi
LOG:  Metadata fresh for importlib.metadata: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/metadata/__init__.pyi
LOG:  Metadata fresh for importlib.readers: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/readers.pyi
LOG:  Metadata fresh for asyncio.coroutines: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/coroutines.pyi
LOG:  Metadata fresh for asyncio.exceptions: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/exceptions.pyi
LOG:  Metadata fresh for asyncio.locks: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/locks.pyi
LOG:  Metadata fresh for asyncio.queues: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/queues.pyi
LOG:  Metadata fresh for asyncio.runners: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/runners.pyi
LOG:  Metadata fresh for asyncio.streams: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/streams.pyi
LOG:  Metadata fresh for asyncio.subprocess: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/subprocess.pyi
LOG:  Metadata fresh for asyncio.threads: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/threads.pyi
LOG:  Metadata fresh for asyncio.graph: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/graph.pyi
LOG:  Metadata fresh for asyncio.taskgroups: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/taskgroups.pyi
LOG:  Metadata fresh for asyncio.timeouts: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/timeouts.pyi
LOG:  Metadata fresh for asyncio.unix_events: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/unix_events.pyi
LOG:  Metadata fresh for concurrent: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/__init__.pyi
LOG:  Metadata fresh for concurrent.futures._base: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/futures/_base.pyi
LOG:  Metadata fresh for concurrent.futures.process: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/futures/process.pyi
LOG:  Metadata fresh for concurrent.futures.thread: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/futures/thread.pyi
LOG:  Metadata fresh for concurrent.futures.interpreter: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/futures/interpreter.pyi
LOG:  Metadata fresh for asyncio.base_futures: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/base_futures.pyi
LOG:  Metadata fresh for _ssl: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_ssl.pyi
LOG:  Metadata fresh for _socket: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_socket.pyi
LOG:  Metadata fresh for importlib.metadata._meta: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/metadata/_meta.pyi
LOG:  Metadata fresh for email.message: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/message.pyi
LOG:  Metadata fresh for importlib.resources.abc: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/resources/abc.pyi
LOG:  Metadata fresh for importlib.resources: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/resources/__init__.pyi
LOG:  Metadata fresh for zipimport: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/zipimport.pyi
LOG:  Metadata fresh for asyncio.mixins: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/mixins.pyi
LOG:  Metadata fresh for asyncio.selector_events: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/asyncio/selector_events.pyi
LOG:  Metadata fresh for multiprocessing.connection: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/connection.pyi
LOG:  Metadata fresh for multiprocessing.context: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/context.pyi
LOG:  Metadata fresh for multiprocessing.queues: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/queues.pyi
LOG:  Metadata fresh for weakref: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/weakref.pyi
LOG:  Metadata fresh for queue: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/queue.pyi
LOG:  Metadata fresh for concurrent.interpreters: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/interpreters/__init__.pyi
LOG:  Metadata fresh for email: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/__init__.pyi
LOG:  Metadata fresh for email.charset: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/charset.pyi
LOG:  Metadata fresh for email.contentmanager: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/contentmanager.pyi
LOG:  Metadata fresh for email.errors: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/errors.pyi
LOG:  Metadata fresh for email.policy: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/policy.pyi
LOG:  Metadata fresh for importlib.resources._common: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/resources/_common.pyi
LOG:  Metadata fresh for importlib.resources._functional: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/importlib/resources/_functional.pyi
LOG:  Metadata fresh for selectors: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/selectors.pyi
LOG:  Metadata fresh for multiprocessing: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/__init__.pyi
LOG:  Metadata fresh for multiprocessing.popen_fork: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/popen_fork.pyi
LOG:  Metadata fresh for multiprocessing.popen_forkserver: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/popen_forkserver.pyi
LOG:  Metadata fresh for multiprocessing.popen_spawn_posix: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/popen_spawn_posix.pyi
LOG:  Metadata fresh for multiprocessing.popen_spawn_win32: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/popen_spawn_win32.pyi
LOG:  Metadata fresh for multiprocessing.synchronize: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/synchronize.pyi
LOG:  Metadata fresh for multiprocessing.managers: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/managers.pyi
LOG:  Metadata fresh for multiprocessing.pool: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/pool.pyi
LOG:  Metadata fresh for multiprocessing.process: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/process.pyi
LOG:  Metadata fresh for multiprocessing.sharedctypes: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/sharedctypes.pyi
LOG:  Metadata fresh for _weakref: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_weakref.pyi
LOG:  Metadata fresh for _weakrefset: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_weakrefset.pyi
LOG:  Metadata fresh for _queue: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_queue.pyi
LOG:  Metadata fresh for concurrent.interpreters._queues: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/interpreters/_queues.pyi
LOG:  Metadata fresh for _interpreters: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_interpreters.pyi
LOG:  Metadata fresh for email._policybase: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/_policybase.pyi
LOG:  Metadata fresh for multiprocessing.reduction: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/reduction.pyi
LOG:  Metadata fresh for multiprocessing.spawn: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/spawn.pyi
LOG:  Metadata fresh for multiprocessing.util: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/util.pyi
LOG:  Metadata fresh for multiprocessing.shared_memory: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/multiprocessing/shared_memory.pyi
LOG:  Metadata fresh for concurrent.interpreters._crossinterp: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/concurrent/interpreters/_crossinterp.pyi
LOG:  Metadata fresh for _interpqueues: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_interpqueues.pyi
LOG:  Metadata fresh for email.header: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/email/header.pyi
LOG:  Metadata fresh for pickle: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/pickle.pyi
LOG:  Metadata fresh for _pickle: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/_pickle.pyi
LOG:  Metadata fresh for copyreg: file /home/bandela/.cache/rattler/cache/cached-envs-v0/mypy-4caf4e062ac41667/lib/python3.14/site-packages/mypy/typeshed/stdlib/copyreg.pyi
LOG:  Loaded graph with 292 nodes (0.015 sec)
LOG:  Found 83 SCCs; largest has 104 nodes
LOG:  Scheduling SCC singleton (myproblem) as inherently stale
LOG:  Processing 81 fresh SCCs
myproblem.py:3: error: "TypeAliasType" not callable  [operator]
LOG:  Writing myproblem myproblem.py myproblem.meta.ff myproblem.data.ff
LOG:  Cached module myproblem has changed interface
LOG:  Build finished in 0.064 seconds with 291 modules, and 1 errors
```

</details>

### Additional typing packages.

Not that I'm aware of

--- a/numpy/ma/core.pyi
+++ b/numpy/ma/core.pyi
@@ -2854,7 +2854,8 @@ class MaskedConstant(MaskedArray[tuple[()], dtype[float64]]):
 masked: Final[MaskedConstant] = ...
 masked_singleton: Final[MaskedConstant] = ...
 
-type masked_array = MaskedArray
+# this should NOT be a (PEP 695) type alias, see https://github.com/numpy/numpy/issues/31737
+masked_array = MaskedArray
 
 # keep in sync with `MaskedArray.__new__`
 @overload

--- a/numpy/typing/tests/data/pass/ma.py
+++ b/numpy/typing/tests/data/pass/ma.py
@@ -7,8 +7,6 @@ from numpy._typing import _Shape
 
 type MaskedArray[ScalarT: np.generic] = np.ma.MaskedArray[_Shape, np.dtype[ScalarT]]
 
-# mypy: disable-error-code=no-untyped-call
-
 MAR_b: MaskedArray[np.bool] = np.ma.MaskedArray([True])
 MAR_u: MaskedArray[np.uint32] = np.ma.MaskedArray([1], dtype=np.uint32)
 MAR_i: MaskedArray[np.int64] = np.ma.MaskedArray([1])
@@ -196,3 +194,6 @@ MAR_c **= AR_LIKE_u
 MAR_c **= AR_LIKE_i
 MAR_c **= AR_LIKE_f
 MAR_c **= AR_LIKE_c
+
+# https://github.com/numpy/numpy/issues/31737
+np.ma.masked_array([1])

